src/main/resources/data.sql

```diff
@@ -14,3 +14,75 @@ insert into airport values
 ('ATW', 'Appleton International Airport','Appleton',  'USA'), 
 ('ATL',  'Hartsfield-Jackson Atlanta International Airport' , 'Atlanta ', 'USA'), 
 ('ACY', 'Atlantic City International Airport', 'Atlantic City', 'USA') ON CONFLICT (airport_id) DO NOTHING;
+
+insert into flight values
+('2345', '4', 'A20', '1', 'APW', 'CAK', '11', 'on time', '250'),
+('3456', '3', 'B30', '2', 'AKL', 'AKC', '13', 'delayed', '300'),
+('4567', '3', 'C40', '2', 'ASG', 'ABQ', '15', 'cancelled', '350'),
+('5678', '4', 'D15', '1', 'AAU', 'ANC', '17', 'delayed', '400.50'),
+('6789', '5', 'C17', '2', 'AMZ', 'ATW', '19', 'on time', '375') ON CONFLICT (flight_number) DO NOTHING;
+
+insert into aircraft values
+('1380000', '116000', '2345', '7250', 'Boeing 757-200PF'),
```

Comment:
These records won't be inserted correctly since the primary keys aren't correct.